docker/m3dbnode/Dockerfile-setcap

```diff
@@ -0,0 +1,34 @@
+# stage 1: build
+FROM golang:1.12-alpine AS builder
+LABEL maintainer="The M3DB Authors <m3db@googlegroups.com>"
+
+# Install Glide
+RUN apk add --update glide git make bash
+
+# Add source code
+RUN mkdir -p /go/src/github.com/m3db/m3
+ADD . /go/src/github.com/m3db/m3
+
+# Build m3dbnode binary
+RUN cd /go/src/github.com/m3db/m3/ && \
+    git submodule update --init      && \
+    make m3dbnode-linux-amd64
+
+# Stage 2: lightweight "release"
+FROM alpine:latest
+LABEL maintainer="The M3DB Authors <m3db@googlegroups.com>"
+
+EXPOSE 2379/tcp 2380/tcp 7201/tcp 7203/tcp 9000-9004/tcp
+
+COPY --from=builder /go/src/github.com/m3db/m3/src/dbnode/config/m3dbnode-local-etcd.yml /etc/m3dbnode/m3dbnode.yml
+COPY --from=builder /go/src/github.com/m3db/m3/bin/m3dbnode \
+  /go/src/github.com/m3db/m3/scripts/m3dbnode_bootstrapped.sh \
+  /bin/
+
+# Use setcap to set +e "effective" and +p "permitted" to adjust the SYS_RESOURCE
+# so the process can raise the hard file limit with setrlimit.
+RUN apk add --no-cache curl jq libcap && \
```

Comment:
Do we need curl and jq?  I'm fine to leave it out and add on login (i.e. just apk add when you start a shell session), or if you feel we should (which I'm fine with too), should we putting it in the other base image too?

Reply:
We only need it for M3DB for the `m3dbnode_bootstrapped.sh` script. When we deprecate that though we can just remove this entirely